lib/MusicBrainz/Server/Entity/Editor.pm

```diff
@@ -104,6 +104,14 @@ sub is_editing_enabled {
     (shift->privileges & $EDITING_DISABLED_FLAG) == 0;
 }
 
+sub is_voting_enabled {
```

Comment:
Well, in the same case as on JS - if we want to know if someone just cannot vote at all (for example, to give someone an error like "you don't have voting rights" vs "you cannot vote on this edit"). The main idea was to keep the parity with the JS... although see the other comment.

Reply:
Do you think we should just make the privileges isVotingEnabled also be able to look at the email confirmation then? I was going to do that, but second-guessed myself because, well, the page is called privileges and the flow types are for privs only right now.

Reply:
I guess the reason it feels weird is because voting isn't a distinct privilege at the moment (it's calculated from other privileges + having a confirmed email).

Maybe we should rename `is_voting_enabled` to `may_vote` on the Perl side and merge `isVotingEnabled` into `editorMayVote` on the JS side? Then I think they'd be named consistently at least.